### PR TITLE
DetailList: Add aria-rowcount and aria-colcount

### PR DESCRIPTION
**Pull request by petdun2519**
#### Pull request checklist

- Addresses an issue (hopefully) filed by accessibility vendors doing a test pass

#### Description of changes

Office fabric grid did not have some aria properties correctly filled, I am attempting to add those properties. Still trying to sync with a broader accessibility champ to make sure my fix will fix the issues the vendors filed.

#### Focus areas to test

(optional)

### CODE REVIEW

File: common/changes/users-petdun-gridAriaProperties_2017-05-01-21-52.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added some aria properties onto components of details list",
```
Comment (cschleiden) [inline common/changes/users-petdun-gridAriaProperties_2017-05-01-21-52.json]: Please use `DetailsList: ... `

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx [hunk 2]
```diff
@@ -247,7 +247,7 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
         data-is-scrollable='false'
         aria-label={ ariaLabel }
         { ...(shouldApplyApplicationRole ? { role: 'application' } : {}) }>
-        <div role='grid' aria-label={ ariaLabelForGrid }>
+        <div role='grid' aria-label={ ariaLabelForGrid } aria-rowcount={ items.length } aria-colcount={ adjustedColumns.length }>
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx]: you want to guard against items being null

Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx]: Probably adjustedColumns, too

File: common/changes/users-petdun-gridAriaProperties_2017-05-01-21-52.json (hunk 1 above)
Comment (petdun2519) [inline common/changes/users-petdun-gridAriaProperties_2017-05-01-21-52.json]: would it track that?

File: packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx [hunk 3]
```diff
@@ -247,7 +247,7 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
         data-is-scrollable='false'
         aria-label={ ariaLabel }
         { ...(shouldApplyApplicationRole ? { role: 'application' } : {}) }>
-        <div role='grid' aria-label={ ariaLabelForGrid }>
+        <div role='grid' aria-label={ ariaLabelForGrid } aria-rowcount={ items ? items.length : 0 } aria-colcount={ adjustedColumns.length }>
```
Comment (cschleiden) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx]: Is `adjustedColumns` always set?

Comment (petdun2519) [inline packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx]: I'll add a null guard there too whoops sorry

**Pull request merged**